geomstats/geometry/functions.py

```diff
@@ -0,0 +1,245 @@
+"""Module for function spaces as geometric objects"""
+
+import math
+
+import numpy as np
+
+import geomstats.backend as gs
+from geomstats.geometry.manifold import Manifold
+from geomstats.geometry.riemannian_metric import RiemannianMetric
+
+
+class SinfSpaceMetric(RiemannianMetric):
```

Comment:
Could we use a more explicit class name? Sinf refers to the notation and does not read well if you don't know what it stands for.

Reply:
Thanks, I am calling this a `HilbertSphere` in the updated code